pkg/machines/libvirt-dbus.js

```diff
@@ -1393,6 +1393,18 @@ export function storagePoolUndefine(connectionName, objPath) {
     return call(connectionName, objPath, 'org.libvirt.StoragePool', 'Undefine', [], TIMEOUT);
 }
 
+export function storageVolumeCreate(connectionName, poolName, volName, size, format) {
+    const volXmlDesc = getVolumeXML(volName, size, format);
+
+    return call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'StoragePoolLookupByName', [poolName], TIMEOUT)
+            .then((path) => {
```

Comment:
nitpick; no need for parenthesis here, I tend to skip them if there is only one arg